components/notifications.js

```diff
@@ -339,6 +339,11 @@ function InvoicePaid ({ n }) {
 
     if (id) payerSig += id
   }
+  return payerSig
+}
+
+function InvoicePaid ({ n }) {
+  const payerSig = getPayerSig(n.invoice.lud18Data)
```

Comment:
For nonP2P at least payer sig is not set for me in dev (I used all lnurl fields): 

<img width="422" alt="Screenshot 2024-11-11 at 5 58 03 PM" src="https://github.com/user-attachments/assets/ccef6c9a-8274-4907-9aed-9c228ce6d1ad">

Reply:
right, i wasn't passing the lud18 data to the action. Fixed now